solutions/swb-app/src/accountRoutes.ts

```diff
@@ -20,7 +20,13 @@ import { wrapAsync } from './errorHandlers';
 import { processValidatorResult } from './validatorHelper';
 
 export function setUpAccountRoutes(router: Router, account: HostingAccountService): void {
-  // Provision
+  router.get(
+    '/aws-accounts/:id',
+    wrapAsync(async (req: Request, res: Response) => {
+      res.send(await account.get(req.params.id));
```

Comment:
What do we get if account not found? undefined?

Reply:
Account IDs that don't match any account currently return a `403 User not authorized`. We could return a 404, but 403s are nice because it prevents anyone from spamming UUIDs to find out which ones are valid and which ones aren't: you will only see the data you're authorized to see.

Reply:
Idk if thats _why_ we're returning a 403, but its a nice security trick 🤷 .

Reply:
I guess something might be broken. I'd expect 404: https://github.com/aws-solutions/solution-spark-on-aws/pull/560/files#diff-896fb3fe131fd2efa124d3ab55693f1ddb4133831a74e07b641617ec0d812bccR199 . Could you please double check?

Reply:
@sevu1 We've discovered an interesting bug! When you put in an invalid UUID as the account id, a 403 is returned, but if you put in a valid UUID that does not match any accounts, it returns a 404! @nguyen102 I'm guessing this isn't the expected behavior?

Reply:
I'm going to merge the change since the bug belongs to the routing code and not my changes .I'm guessing we'll want to file a ticket to change it so that any invalid id returns a 404? (or a 403 if we don't want to let attackers try and guess UUIDs?)

Reply:
It could be 400 as well